frontend/src/pages/ApprovedActivityReport/helpers.js

```diff
@@ -78,7 +78,7 @@ export function mapAttachments(attachments) {
     );
   }
 
-  return [];
+  return 'None provided';
```

Comment:
Just want to double check that the string being returned here will work as we were returning an array before.

Not sure if its expecting `['None provided']`

Reply:
Yep, double-checked it locally and returning a string instead of an empty array works.